main.py

```diff
@@ -18,15 +18,37 @@
 from flask import Flask, render_template, redirect, url_for, request, abort, Response, jsonify
 from db_operations import *
 from forms import AdoptionForm, CreateListingForm
+from listings import Listing
+
 
 app = Flask(__name__)
 app.config['SECRET_KEY'] = os.urandom(32)
 
 
-@app.route('/')
+@app.route('/', methods=['GET', 'POST'])
 def root():
-    listings = get_listings()
-    return render_template('main.html', listings=listings)
+    if request.method == 'GET':
+        listings = get_listings()
+        form = CreateListingForm()
+        if form.validate_on_submit():
+            print(form.email)
```

Comment:
change the print statements to log statements

Reply:
Removed those print statements (for debugging purposes)